app/src/main/index.js

```diff
@@ -312,10 +314,14 @@ const openPrefsWindow = () => {
 
   prefsWindow.on('close', () => {
     prefsWindow = undefined;
+    track('preferences/closed');
```

Comment:
I don't see how this is useful. We already know it was opened. Of course the user is going to close it too. Unless I'm missing something.